Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -0,0 +1,55 @@
+Import-VstsLocStrings -LiteralPath $PSScriptRoot/module.json
+
+function Add-Tls12InSession {
+    [CmdletBinding()]
+    param()
+
+    try {
+        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
+            [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]3072
+            Write-Host (Get-VstsLocString -Key TLS12AddedInSession)
+        }
+        else {
+            Write-Verbose 'TLS 1.2 already present in session.'
+        }
+    }
+    catch {
+        Write-VstsTaskError "Failed to add TLS 1.2 in session: $_.Exception.Message"
+    }
+}
+
+function Assert-TlsError {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory=$true)] $exception 
```

Comment:
since exception is a mandatory param here, the null check that follows will never be true. additionally it will throw when this function is called with null. allow null value here.